example_site/src/components/ConfigurationControls.js

```diff
@@ -281,64 +284,80 @@ export default function ConfigurationControls({ isOpen }) {
+                  className="form-input"
+                  name="unsavedChangesModalTitle"
+                  defaultValue="Exit Editor"
+                  ref={register}
+            />
+            
+            <label
+              className="form-label configuration-column-3"
+              htmlFor="unsavedChangesModalBody"
+            >
+              Modal Body Text
+            </label>
 
-          <label
-            className="form-label configuration-column-3"
-            htmlFor="unsavedChangesModalBody"
-          >
-            Modal Body Text
-          </label>
-          <input
-            type="text"
-            className="form-input"
-            name="unsavedChangesModalBody"
-            defaultValue="Your video has unsaved edits. Are you sure you want to leave?"
-            ref={register}
-          />
+            <input
+              type="text"
+              className="form-input"
+              name="unsavedChangesModalBody"
+              defaultValue="Your video has unsaved edits. Are you sure you want to leave?"
+              ref={register}
+            />
 
-          <label
-            className="form-label configuration-column-3"
-            htmlFor="unsavedChangesModalConfirmButton"
-          >
-            Modal Confirmation Button Label
-          </label>
-          <input
-            type="text"
-            className="form-input"
-            name="unsavedChangesModalConfirmButton"
-            defaultValue="Exit Editor"
-            ref={register}
-          />
+            <label
+              className="form-label configuration-column-3"
+              htmlFor="unsavedChangesModalConfirmButton"
+            >
+              Modal Confirmation Button Label
+            </label>  
 
-          <label
-            className="form-label configuration-column-3"
-            htmlFor="unsavedChangesModalCancelButton"
-          >
-            Modal Cancel Button Label
-          </label>
-          <input
-            type="text"
-            className="form-input"
-            name="unsavedChangesModalCancelButton"
-            defaultValue="Cancel"
-            ref={register}
-          />
-        </div>
+            <input
+              type="text"
+              className="form-input"
+              name="unsavedChangesModalConfirmButton"
+              defaultValue="Exit Editor"
+              ref={register}
+            />
+
+            <label
+              className="form-label configuration-column-3"
+              htmlFor="unsavedChangesModalCancelButton"
+            >
+              Modal Cancel Button Label
+            </label>
 
+            <input
+              type="text"
+              className="form-input"
+              name="unsavedChangesModalCancelButton"
+              defaultValue="Cancel"
+              ref={register}
+            />
+          </div>
+        </div>
+      </div>
+      
+      <div>
+        <h1></h1>
+        <h2></h2>
+        <br></br>
```

Comment:
What are these additional elements for?

Reply:
Intended for spacing, made changes so top padding (CSS) is used instead!